Makefile

```diff
@@ -69,4 +69,4 @@ all:
 
 install:
 	install -m 644 pc-boot.img pc-core.img shim.efi.signed grubx64.efi $(DESTDIR)/
-	install -m 644 grub.cfg-recovery grub.cfg-normal grub.conf $(DESTDIR)/
+	install -m 644 grub.cfg-recovery grub.cfg-normal grub-recovery.conf grub.conf $(DESTDIR)/
```

Comment:
I wonder if we really need two real files (one of which is unused), and two symlinks, both pointing at grub.cfg-recovery =)

I guess this is a todo to clean up. Originally grub.conf was meant to point at one or the other. Or something like that?!

Reply:
I also wondered if the files could be renamed instead of using symlinks (or maybe the rest of the buildsystem expects the original names?). We can check this with @mvo5.